uber/templates/groups/history.html

```diff
@@ -2,7 +2,12 @@
 {% block title %}Group History - {{ group.name }}{% endblock %}
 {% block content %}
 
-{% nav_menu group "form?id={id}" "Group Page" True "history?id={id}" "History" True "index" "Return to Group List" True %}
+{{ macros.nav_menu(
```

Comment:
i think Eli said this in a previous comment, and we can ignore this for now, however, I think nav_menu() eventually needs to die, at least in its current form.

he had mentioned it was from an ancient version of django that had some restrictions on how you could do something like this.

for right now, if it's working, ignore it.